package/MDAnalysis/coordinates/TNG.py

```diff
@@ -0,0 +1,437 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+TNG trajectory files --- :mod:`MDAnalysis.coordinates.TNG`
+==========================================================
+
+
+The TNG format is a format used in `GROMACS`_ for storage of trajectory and
+topology information. The TNG format allows a wide range of compression
+algorithms and unlike the compressed XTC format can also store velocities
+forces in addition to positions.
+
+The classes in this module are based on the `pytng`_ package for reading TNG files
+The reader is directed to the `documentation`_ for further reading about how pytng
+works under the hood.
+
+In addition to particle-dependent trajectory information like positions,
+forces and velocities, the TNG format can store trajectory metadata and 
+other arbitrary time dependent data. Additional information can range from
+the virial and pressure components to the molecular topology of the system.
+This is enabled by a block based system in which binary flags indicate the
+presence or absence of various data blocks. The structure of a TNG file is 
+provided  in the TNG `specification`_. The TNG `paper`_ and the
+pytng `documentation`_ are also good resources. The user is encouraged to read
+the full list of `blocks`_ to understand the full power of the TNG format.
+
+
+
+Notes
+-----
+Currently there is only a Reader for TNG files and no writer. This will depend
```

Comment:
reader capitalised but writer isn't? should do both of either capitalised or not